newsletter.md

```diff
@@ -0,0 +1,14 @@
+
+During the past few months, the team has been working hard to make new features available in the OpenPaaS MVP release.
+
+
+The team implemented JMAP sharing, allowing one to share a folder with another user of the domain. The sharee gets the right, if allowed, to read, mark delete and moves emails. This is especially useful to delegate the management of your emails to someone else. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting JMAP sharing feature in INBOX.
+
+
+We also implemented drafts. The email you compose can now be saved in INBOX, reopened, re-edited. Moving them to the Outbox mailbox will actually trigger the sending. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting the draft feature in INBOX.
+
+
+After implementing these features, we were also involved in the "polish" sprint (see above?), allowing us to fix some little but annoying issues.
+
+
+In the coming months, the team will be focused on delivering a highly reliable software. We will move toward latest states of the JMAP specifications and implement JMAP instant notifications (push), make James more distributed, and improve distributed safety.
```

Comment:
I don't remember that we spoke on incoming works in the newsletter.

Reply:
I consider saying what we intend to do to give a bit of visibility should be a general good practice...

Reply:
I agree with you, but we never do that before and its a newsletter, not a forecast.
BTW, as James will not be each months in the OP newsletter, it could be great to speak on what we are gonna work.

Reply:
Thank you @chibenwa for the James newsletter part. I have several remarks:
1- it would be a good idea to discuss with Inbox team about your respective content for the December newsletter in order not to be redundant.
2- the content is a bit light compared to the 20 expected lines.
3- announcing what will be done is OK because it invites our readers to follow our upcoming newsletters and we already did it.

Reply:
@aduprat Often when I read newsletter I heard about project considering the futur. So I do not see any problem about speaking about what come next

Reply:
Hi @Laura-Royet, pasted in medium the article is 16 lines, which is not so for from the 20 expected :)
Do you have some advice about what we could add or explain better to add a little content?

Reply:
@rouazana : you're right the number of lines is ok.